lincs/resources/views/CustomGCT.html

```diff
@@ -7,9 +7,13 @@
         var url = LABKEY.ActionURL.buildURL("lincs", "createCustomGCT", LABKEY.ActionURL.getContainer());
         window.location = url;
     }
+    LABKEY.Utils.onReady(function()
+    {
+        document.getElementById("customGctButton").onclick = function() { showCustomGCTForm(); };
```

Comment:
This is already merged and no need to change, but:

- By convention, we usually do something like: `document.getElementById("customGctButton")['onclick'] = function()...`. This drops the migrated handlers from text searches looking for typical inline handler patterns ("onclick = ").
- Since there are no parameters, `= showCustomGCTForm;` should work as well.